### ISSUE TITLE
Crash on asking AppleExerciseTime Permission 

### ISSUE THREAD

**Issue by ManetJun** [agencyenterprise/react-native-health#22]
**Describe the bug**
Initiating connection to HealthKit app, adding AppleExerciseTime permission alone cause a crash.

**To Reproduce**
1. Initiate permission request for AppleExerciseTime alone(see code snippet below).
2. App will crash

**Expected behavior**
Should not crash.

**Screenshots**
<img width="273" alt="Screenshot 2020-11-05 at 11 32 05 AM" src="https://user-images.githubusercontent.com/59823120/98194528-94dd9780-1f5a-11eb-835d-4503d2d5d9ed.png">

**Smartphone:**
 - Device: iPhoneXs
 - OS: iOS 12.2

**Additional context**
Tried this in simulator though and it crash.
Other permissions like StepCount, SleepAnalysis it's working fine and can able to pull the data.

The RN library has a function to retrieve exercise time(AppleHealthKit.getAppleExerciseTime). 
However, looks like in native level(OC), 'HKQuantityTypeIdentifierAppleExerciseTime' permission was not included in this function: `- (nullable HKObjectType *)getReadPermFromText:(nonnull NSString*)key` in the `RCTAppleHealthKit+TypesAndPermissions.m` class.

**Code snippet**
```
const permissions = {
    permissions: {
      read: [
        AppleHealthKit.Constants.Permissions.AppleExerciseTime
      ]
    }
  } as HealthKitPermissions

AppleHealthKit.initHealthKit(this.permissions, (error: string) => {})
```

**Comment by lucaspbordignon** [comment]
Hey @ManetJun!

First of all, sorry for the delay in the response, couldn't give the proper attention to the project in the past weeks

Looking into the code snippet you provided, seems that the root cause of it is really the lack of the `HKQuantityTypeIdentifierAppleExerciseTime` permission. Looking into the native level implementation we don't have the addition of the permission when a user requests for it.

I'll create a new PR to fix that

**Comment by lucaspbordignon** [comment]
It seems that the permission was removed in 906ba45, as it is available only for iOS 9.3+
